install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1699,13 +1704,204 @@ dashboards:
+  podSecurityContext: {}
+
+  # -- Annotations to be added to envoy pods
+  podAnnotations: {}
+
+  # -- Labels to be added to envoy pods
+  podLabels: {}
+
+  # -- Envoy resource limits & requests
+  # ref: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+  resources: {}
+    # limits:
+    #   cpu: 4000m
+    #   memory: 4Gi
+    # requests:
+    #   cpu: 100m
+    #   memory: 512Mi
+
+  startupProbe:
+    # -- failure threshold of startup probe.
+    # 105 x 2s translates to the old behaviour of the readiness probe (120s delay + 30 x 3s)
+    failureThreshold: 105
+    # -- interval between checks of the startup probe
+    periodSeconds: 2
+  livenessProbe:
+    # -- failure threshold of liveness probe
+    failureThreshold: 10
+    # -- interval between checks of the liveness probe
+    periodSeconds: 30
+  readinessProbe:
+    # -- failure threshold of readiness probe
+    failureThreshold: 3
+    # -- interval between checks of the readiness probe
+    periodSeconds: 30
+
+  securityContext:
+    # -- User to run the pod with
+    # runAsUser: 0
+    # -- Run the pod with elevated privileges
+    privileged: false
+    # -- SELinux options for the `cilium-envoy` container
+    seLinuxOptions:
+      level: 's0'
+      # Running with spc_t since we have removed the privileged mode.
+      # Users can change it to a different type as long as they have the
+      # type available on the system.
+      type: 'spc_t'
+    capabilities:
+      # -- Capabilities for the `cilium-envoy` container
+      envoy:
+        # Used since cilium proxy uses setting IPPROTO_IP/IP_TRANSPARENT
+        - NET_ADMIN
+        # We need it for now but might not need it for >= 5.11 specially
+        # for the 'SYS_RESOURCE'.
+        # In >= 5.8 there's already BPF and PERMON capabilities
+        - SYS_ADMIN
+        # Both PERFMON and BPF requires kernel 5.8, container runtime
+        # cri-o >= v1.22.0 or containerd >= v1.5.0.
+        # If available, SYS_ADMIN can be removed.
+        #- PERFMON
+        #- BPF
+
+  # -- Affinity for cilium-envoy.
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - topologyKey: kubernetes.io/hostname
+        labelSelector:
+          matchLabels:
+            k8s-app: cilium-envoy
+
+  # -- Node selector for cilium-envoy.
+  nodeSelector:
+    kubernetes.io/os: linux
+
+  # -- Node tolerations for envoy scheduling to nodes with taints
+  # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
+  tolerations:
+  - operator: Exists
+    # - key: "key"
+    #   operator: "Equal|Exists"
+    #   value: "value"
+    #   effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
+
+  # -- The priority class to use for cilium-envoy.
+  priorityClassName: ""
+
+  # -- DNS policy for Cilium envoy pods.
+  # Ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
+  dnsPolicy: ""
```

Comment:
likewise, should be `~`

Reply:
@squeed there's no default for `dnsPolicy` in the envoy daemonset

Reply:
right, which is why this should be `~`. Otherwise `""` is documented as the default.

Reply:
@squeed ok - changed here as well :+1: